### PR TITLE
Voice Broadcast recording pip

### PR DESCRIPTION
**Pull request by weeman1337**
Part of https://github.com/vector-im/element-web/issues/23282

- Simple initial voice broadcast recording PiP implementation
- Does not handle call concurrency in a great way, yet
- Enough for the first feature delivery

image

## Checklist

* [x] Tests written for new code (and old code if feasible)
* [ ] Linter and other CI checks pass
* [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Review by robintown — COMMENTED** [review]
Just a couple nits, otherwise looks good!

File: src/components/views/voip/PipView.tsx [hunk 1]
```diff
@@ -367,3 +383,27 @@ export default class PipView extends React.Component<IProps, IState> {
...
+        },
+    );
+
+    return <PipView
+        voiceBroadcastRecording={voiceBroadcastRecording}
+        {...props}
+    />;
+};
+
+export default PipViewHOC;
```
[the remark this answers is not in the document]
Comment (weeman1337) [reply src/components/views/voip/PipView.tsx:409]: Yes. But I don't want to change too much here. This is changing the current default export to the HOC.

[the remark this answers is not in the document]
Comment (robintown) [reply src/components/views/voip/PipView.tsx:409]: Got it, that's fine then

File: src/voice-broadcast/components/atoms/StopButton.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    onClick: () => void;
+}
+
+export const StopButton: React.FC<Props> = ({
+    onClick,
+}) => {
+    return <AccessibleButton
+        className="mx_BroadcastPlaybackControlButton"
+        onClick={onClick}
+        aria-label={_t(" stop voice broadcast")}
```
[the remark this answers is not in the document]
Comment (weeman1337) [reply src/voice-broadcast/components/atoms/StopButton.tsx:33]: Good catch :+1:

**Review by robintown — APPROVED** [review]

**Pull request merged**